### ISSUE TITLE
For automation purposes, write authentication token to stdout and informational messages to stderr

### ISSUE THREAD

**Issue by agross** [mkb79/audible-cli#42]
I would like to automate audible-cli, specifically getting the activation bytes with `audible activation-bytes`.

Running this in a script after initial `quickstart` prints the following:

```
$ audible activation-bytes
Activation bytes not found in auth file. Fetching online.
abcdefgh
```

I'm interested in the `abcdefgh` part, but that is not easily assignable to a shell variable because `Activation bytes not found in auth file. Fetching online.` clutters stdout:

```sh
bytes=$(audible activation-bytes)

# bytes contains the token and the extra output!
```

The preferred way of solving this would be to use stderr for informational messages and stdout for the results, so I can use the CLI call like this:

```sh
bytes=$(audible activation-bytes 2>/dev/null)

# bytes contains only the token
```

It would be great if this could be added!

**Comment by mkb79** [comment]
You can use the command with the `--save` option. This will save the ab to auth file and next time the ab only will be printed out!

**Comment by agross** [comment]
Yes, I have found that as well as a workaround. I'll leave it up to you to close this issue.

**Comment by mkb79** [comment]
The preferred way is to fetch the activation bytes only once online and then save them. So I think I will remove the `--save` option and let the command save them the first time and then reuse it from the auth file.

**Comment by agross** [comment]
Works with the dev version!

**Comment by mkb79** [comment]
Good to hear that it works. I published v0.0.5 to PyPI which contains all the changes.

**Comment by agross** [comment]
Thank you!
